test/integration/au/org/emii/aatams/report/resources/testExecuteInstallationStationNoFilter.expected.csv

```diff
@@ -1,24 +1,22 @@
-,,AATAMS Installation Stations,,,,,,,,,,,,,
+,,[ THIS LINE WILL BE STRIPPED],,,,,,
 ,Parameters,,,,,,,,,,,,,,
 ,user:,,,,,,Jon Burgess,,,,,,,,
-,,,,configuration,,,,station name,,,latitude,longitude,active,,
+,,,configuration,,,,,station name,,,latitude,longitude,active,,
 ,project:,,,,Seal Count,,,,organisations:,CSIRO (CMAR),,,,,
-,,,installation:,,,Bondi Line,,,,,,,,,
-,,,,CURTAIN,,,,Bondi SW1,,,30.1234,30.1234,yes,,
-,,,,CURTAIN,,,,Bondi SW2,,,-10.1234,-10.1234,yes,,
-,,,,CURTAIN,,,,Bondi SW3,,,10.1234,10.1234,no,,
-,,,installation:,,,Heron Island ,,,,,,,,,
-,,,,CURTAIN,,,,Heron S1,,,-42.3,12.34,yes,,
-,,,,CURTAIN,,,,Heron S2,,,-20.1234,76.02,yes,,
+,,installation:,,,,Bondi Line,,,,,,,,,
+,,,CURTAIN,,,,,Bondi SW1,,,30.1234,30.1234,yes,,
```

Comment:
@julian1 
The commas still add up to the same amount. It looks like they are moved in position rather than some data having been dropped off.